include/tvm/runtime/packed_func.h

```diff
@@ -549,25 +586,41 @@ class TVMPODValue_ {
     // the frontend while the API expects a float.
     if (type_code_ == kDLInt) {
       return static_cast<double>(value_.v_int64);
+    } else if (auto opt = ThroughObjectRef<double>()) {
+      return opt.value();
+    } else if (auto opt = ThroughObjectRef<int64_t>()) {
+      return opt.value();
```

Comment:
Shouldn't there be `static_cast<double>`, as in line 588?

Reply:
Good catch.  It isn't strictly necessary in either case, due to C++'s implicit conversion of numeric types, but would be good to call attention to it.  Updated to use a `static_cast<double>` here.